docker/Dockerfile

```diff
@@ -123,10 +129,17 @@ RUN set -e; \
         for attempt in $(seq "$DEB_MAX_ATTEMPT" -1 0); do \
             [ "$attempt" -gt 0 ]; \
             sudo DEBIAN_FRONTEND=noninteractive apt-get install -y \
+                $(case "$TARGETARCH" in \
+                   'amd64') \
+                       echo ' \
+                          bazel=5.2.* \
```

Comment:
Don't group, check individually with [ "..." != "_amd64" ] || echo ... and keep them sorted.